patch/1.19.9/lua-resty-core-ngx_meta_lua_module.patch

```diff
@@ -0,0 +1,273 @@
+diff --git lib/resty/core/shdict.lua lib/resty/core/shdict.lua
```

Comment:
We should name the patch with its function `shared_shdict`

Reply:
changed